doc/js-api/remotestorage.rst

```diff
@@ -70,19 +70,19 @@ List of events
 
 ``wire-busy``
 """""""""""""
-   Emitted when a wire request starts
+   Emitted when a network request starts
 
 ``wire-done``
 """""""""""""
-   Emitted when a wire request completes
+   Emitted when a network request completes
 
 ``sync-req-done``
 """""""""""""""""
-   Emitted when a single sync task has finished
+   Emitted when a sync request has finished
```

Comment:
I actually liked the "single" in there. Makes it absolutely clear what is meant.